source/articles/2015/how-to-generate-sales-leads-as-a-freelance-developer.md

```diff
@@ -0,0 +1,223 @@
+---
+title: How to Generate Sales Leads as a Freelance Developer
+description: Using networking and productivity tactics to generate sales interest
+
+author: Adarsh Pandit
+category: consulting
+date: 2015-08-08
+logo: terminal_icon.png
+published: true
+tags: [consulting, sales, leads, networking, content marketing]
+---
+
+
+As I wrote in [this post][1],
+I'm sharing what I know about
+consulting in hopes it's helpful to others.
+
+This post focuses on how to generate sales leads or
+"finding people who are interested in paying you."
+
+Everything here I learned as a freelance
+management consultant,
+while selling millions of dollars of work
+for a medium-sized boutique firm,
+and now as a freelance developer.
+
+[1]: /how-to-run-a-freelance-software-consulting-business
+
+## Preface
+
+It's a weird thing to hire someone else
+and pay them for their time.
+
+The relationship requires trust above all else.
+Once one party loses trust in the other,
+it's best to move on, re-evaluate what happened,
+and find other work.
+As such, everything you do,
+needs to be authentic, honest, and professional,
+including trying to drum up business.
+
+Also it's worth noting these tactics work best
+when you do them very consistently
+for a long time.
+There's no substitute for building good routines
+and doing the work.
+
+Finally, it should be noted
+that I am by no means perfect at sales
+and your mileage may vary.
+
+Okay lets start:
+
+## Networking
+
+### Specific Tactics I Use
+
+* Keep a list of people to stay in touch with.
+* Be sure to follow up regularly. Use tools to automate the process.
+  I like to send a reminder to [`every2months@followup.cc`][2] with
```

Comment:
Followupthen looks like the best option, they're free and their first paid tier is $2/month: https://www.followupthen.com/features